airbyte-integrations/connectors/source-iterable/main_dev.py

```diff
@@ -23,7 +23,7 @@
 
 import sys
 
-from base_python.entrypoint import launch
+from airbyte_cdk.entrypoint import launch
```

Comment:
I think `main_dev.py` should also be renamed to `main.py`, don't forget to update README